prov/hook/trace/src/hook_trace.c

```diff
@@ -48,6 +46,99 @@
 	TRACE_EP(ret, ep, count ? iov[0].iov_base : NULL, \
 		 ofi_total_iov_len(iov, count), addr, tagkey, ignoredata, context)
 
+typedef void (*trace_cq_entry_fn)(const struct fi_provider *prov,
+				const char *func, int line,
+				int count, const void *buf, uint64_t data);
+
+static void
+dummy(const struct fi_provider *prov, const char *func, int line,
+		int count, const void *buf, uint64_t data) { }
```

Comment:
Please move braces to their own lines.  I'd rename to trace_cq_unknown() or noop.
Align second function line one space to the right of the '(', so under the const.  This comment applies to the other functions in this file.